### PR TITLE
Add salesforce soap connector configuration

### PR DESCRIPTION
**Pull request by KasunAratthanage**

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/salesforce-soap-connector/sf-soap-connector-config.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Follow the steps below to import the Salesforce certificate into the EI client keystore:
+
+1. To view the certificate, log in to your Salesforce account in your browser.
+2. Search the **Certificate and Key Management** in the search box.
+
+   <img src="../../../../assets/img/connectors/salesforcesoap-certificste-and-key-management.png" title="salesforcesoap-certificste-and-key-management" width="600" alt="salesforcesoap-certificste-and-key-management"/>  
+    
+3. Export the certificate to the file system.
+4. Import the certificate to the EI client keystore using either the following command or the [EI Management Console](https://docs.wso2.com/display/ESB490/Import+Certificates+for+Keystores):
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/salesforce-soap-connector/sf-soap-connector-config.md:27]: Shall we use a relative link to this doc instead: https://ei.docs.wso2.com/en/7.1.0/micro-integrator/setup/security/importing_ssl_certificate/

File: en/micro-integrator/docs/references/connectors/salesforce-soap-connector/sf-soap-connector-config.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    ```
+    <salesforce.init>
+        <username>MyUsername</username>
+        <password>MyPassword</password>
+        <loginUrl>https://login.salesforce.com/services/Soap/u/42.0</loginUrl>
+        <blocking>false</blocking>
+    </salesforce.init>
+    ```
+
+> **Note**: Secure Vault is supported for [encrypting passwords](https://docs.wso2.com/display/ADMIN44x/Encrypting+Passwords+with+Cipher+Tool). See, Working with Passwords on integrating and using Secure Vault.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/salesforce-soap-connector/sf-soap-connector-config.md:44]: Do we have this in the latest docs? Is it https://ei.docs.wso2.com/en/7.1.0/micro-integrator/setup/security/single_key_encryption/

File: en/micro-integrator/docs/references/connectors/salesforce-soap-connector/sf-soap-connector-config.md (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/salesforce-soap-connector/sf-soap-connector-config.md:27]: Updated.

File: en/micro-integrator/docs/references/connectors/salesforce-soap-connector/sf-soap-connector-config.md (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/salesforce-soap-connector/sf-soap-connector-config.md:44]: yes.Updated.https://ei.docs.wso2.com/en/7.1.0/micro-integrator/setup/security/encrypting_plain_text/

**Pull request merged**